basic.py

```diff
@@ -90,3 +91,33 @@ def eval_(event):
         return eval(input, {"__builtins__": None}, safe_dict)
     except Exception as e:
         return "eval: {0}".format(e)
+
+
+last_events = {}
+
+
+@event
+def sub_event_catcher(event):
+    if event.event_type in ['message', 'cmd']:
+        last_events[event.meta['host']] = event
+
+
+@match(regex=r'^s/(.*?)/(.*?)/([a-z]*)$')
+def sub_match(event, pattern, repl, flags, *args, **kwargs):
```

Comment:
Well it confused me when I first took a look at the code (actually at example) for like 0.8s so... :-D I'd suggest changing it. But it's your call (I have only thought of spelling it out in full: `replacement`).